packages/hooks/user.ts

```diff
@@ -73,11 +73,28 @@ export function useUserMutation() {
     { onSuccess: invalidateCache, onError },
   );
   const closeAccount = useMutation(
-    () =>
-      karrio.graphql.request<update_user>(gqlstr(UPDATE_USER), {
-        data: { is_active: false },
-      }),
-    { onSuccess: invalidateCache, onError },
+    async () => {
+      // Bypass org header by using a direct axios POST (Authorization only)
+      const host = getHost?.();
+      const graphqlUrl = (metadata as any)?.GRAPHQL || url$`${host}/graphql`;
+      const session: any = await getSession();
+      const headers: Record<string, string> = { "content-type": "application/json" };
+      if (session?.accessToken) headers["authorization"] = `Bearer ${session.accessToken}`;
+      if (session?.testMode) headers["x-test-mode"] = "true";
+
+      const body = {
+        query: `mutation update_user($data: UpdateUserInput!) { update_user(input: $data) { user { email } errors { field messages } } }`,

```

Comment:
This should move to the queries.ts file where all the queries and mutations are defined and not hardcoded here. Please add a personal task to do that correction from main as I am going to proceed to merging this as is now.